zksync/sdk/musig/schnorr_musig.py

```diff
@@ -0,0 +1,86 @@
+from ctypes import c_size_t
+from ctypes import c_ubyte
+from typing import List
+
+from zksync.sdk.musig.schnorr_musig_error import SchnorrMusigError
+from zksync.sdk.musig.schnorr_musig_native import STANDARD_ENCODING_LENGTH
+from zksync.sdk.musig.schnorr_musig_native import SchnorrMusigLoader
+from zksync.sdk.musig.schnorr_musig_signer import AggregatedPublicKey
+from zksync.sdk.musig.schnorr_musig_signer import AggregatedPublicKeyPointer
+from zksync.sdk.musig.schnorr_musig_signer import AggregatedSignature
+from zksync.sdk.musig.schnorr_musig_signer import MusigRes
+from zksync.sdk.musig.schnorr_musig_signer import SchnorrMusigSigner
+
+
+class SchnorrMusig:
+
+    def __init__(self) -> None:
+        self.musig = SchnorrMusigLoader.load()
+
+    def create_signer_from_keys(self, public_keys: List[List[int]], position: int) -> SchnorrMusigSigner:
+        encoded_public_keys = []
+        for public_key in public_keys:
+            encoded_public_keys += public_key
+        encoded_public_keys_len = len(encoded_public_keys)
+        signer = self.musig.schnorr_musig_new_signer((c_ubyte * encoded_public_keys_len)(*encoded_public_keys),
+                                                     c_size_t(encoded_public_keys_len), c_size_t(position))
+
+        return SchnorrMusigSigner(self.musig, signer, public_keys[position])
+
+    def create_signer_from_encoded_keys(self, encoded_public_keys: List[int], position: int) -> SchnorrMusigSigner:
+        public_keys_len = len(encoded_public_keys)
+        signer = self.musig.schnorr_musig_new_signer((c_ubyte * public_keys_len)(*encoded_public_keys),
+                                                     c_size_t(public_keys_len), c_size_t(position))
+        start_index = position * STANDARD_ENCODING_LENGTH
+        public_key = encoded_public_keys[start_index: start_index + STANDARD_ENCODING_LENGTH]
+
+        return SchnorrMusigSigner(self.musig, signer, public_key)
+
+    def create_signer_from_key(self, public_key: List[int]) -> SchnorrMusigSigner:
```

Comment:
I can't think of a use-case where this can be useful. Instead, I believe we should support initializing the signer with a list of bytes and perhaps a list of hex strings for more flexibility. The same goes for other interfaces which take bytes as input.